src/checks/one-relearn-move.js

```diff
@@ -0,0 +1,15 @@
+const gameCheck = require('../helpers/gameId.js');
+
+module.exports = {
+  description: 'Pokemon hatched from an egg in gen 6-7 must have at least one relearn move.',
+  field: 'eggMoveName',
+  filter (pkmn) {
+    return !(pkmn.eggLocationId === 0);
```

Comment:
This will filter out everything that isn't from an egg. But I think the check should still get run on non-egg Pokémon from previous generations.

There are a few options here:

* Split this into two different checks (one for pre-gen6 Pokémon, and one for gen 6+ eggs)
* Keep it as a single check, but update the description and this filter